src/main/scala/com/micronautics/concurrent/SynchronizedLruMap.scala

```diff
@@ -0,0 +1,35 @@
+package com.micronautics.concurrent
```

Comment:
I don't think having two different packages is the way to go

Reply:
No reason given

Reply:
It's just good practice not to have two completely different packages inside the same library :+1: